src/Repository/EventRepository.php

```diff
@@ -113,4 +113,149 @@ public function countItemsAdmin()
             ->getQuery()
             ->getSingleScalarResult();
     }
+
+    /**
+     * @param $limit
+     * @param $offset
+     */
+    public function findListAdminToApprove($limit, $offset)
+    {
+        return $this->createQueryBuilder('event')
+            ->andWhere('event.status = :val')
+            ->setParameter('val', Event::STATUS_PENDING)
+            ->orderBy('event.startDate', 'ASC')
+            ->setFirstResult($offset)
+            ->setMaxResults($limit)
+            ->getQuery()
+            ->getResult();
+    }
+
+    /**
+     * @return integer
+     */
+    public function countItemsAdminToApprove()
+    {
+        return $this->createQueryBuilder('event')
+            ->select('count(event.id)')
+            ->andWhere('event.status = :val')
+            ->setParameter('val', Event::STATUS_PENDING)
```

Comment:
@daria-sieroshtan 
That's a good place to use expr for equality, because it saves you from param:
```PHP
->andWhere($qb->expr()->eq('event.status', Event::STATUS_PENDING))
```

Reply:
This change won't add neither simplicity nor briefness